### DISCUSSION TITLE
说个实话win7版的7.8.2还没有7.7.2的稳定

### DISCUSSION THREAD

**Discussion by woodchuckk** [getlantern/lantern-forum-cn#353] [Q&A]
主要问题在于7.8.2之前闪退过，现在虽然抢修没有闪退了，但是不断跳服务器地址这个毛病还存在，而且就算是断了链接，蓝灯图标也还是亮着不是灰色，所以使用者根本不知道自己掉了，非常不稳定，相对来说7.7.2就稳定很多，而且掉了也老老实实图标会变灰。
要是谁看到我这条，可以做个参考，多存几个他们发布的版本，建议蓝灯客服也把蓝灯之前的旧版本再发出来分享一遍，这样也能减少客诉啊

**Comment by MaxMKMax** [comment]
@woodchuckk 嗯嗯. 您的建议挺好的. 我来反馈给他们一下. 之所以没放旧的版本出来是因为之前的版本也有别的问题. 

我来反馈让他们知道问题所在.

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 谢谢啊!

**Reply by woodchuckk** [reply]
票号70990018，你们的DE和JP这俩服务器都不能用，只能用的现在只有SG和US
